README.md

```diff
@@ -1,8 +1,9 @@
 # koku-metrics-operator
 
-[![License: Apache-2.0](https://img.shields.io/github/license/project-koku/koku.svg)](https://www.apache.org/licenses/LICENSE-2.0)
+[![License: Apache-2.0](https://img.shields.io/github/license/project-koku/koku-metrics-operator)](https://github.com/project-koku/koku-metrics-operator/blob/main/LICENSE)
```

Comment:
Let's just remove this badge because Github makes it pretty clear what license a repo has in the UI:
![image](https://github.com/user-attachments/assets/c764c230-a3ce-4a4c-a029-cb20172cdea4)
![image](https://github.com/user-attachments/assets/0a74591b-6d32-4732-b713-522db0849bc7)

So the badge feels super redundant now.

Reply:
updated the doc to drop license badge